features/opennms-gwt-theme/.gitignore

```diff
@@ -1 +0,0 @@
-src/main/webapp/
```

Comment:
Are these files regenerated during each build or was ```src/main/webapp``` added to this .gitignore by mistake?